codegenerator/cli/npm/envio/src/db/Table.res

```diff
@@ -240,6 +240,7 @@ let toSqlParams = (table: table, ~schema, ~pgSchema) => {
           switch field {
           | Field(f) =>
             switch f.fieldType {
+            | Custom(fieldType) if fieldType->Js.String2.startsWith("NUMERIC(") => fieldType
```

Comment:
Can you just add a comment above this when it would start with Numeric? It's obviously an edge case